### PR TITLE
GE: Payments labels and ui

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata
- Custom Labels
  - commonCVV
  - geTextPaymentsSaveNotice
# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: W-040613

**Comment by psadie-sfdo** [comment]
**lurch: attach W-040613

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -350,8 +357,17 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
...
+        const apexException = new ExceptionDataError(error.apexException);
+        this.pageLevelErrorMessageList = [{
+            index: 0,
+            errorMessage: this.CUSTOM_LABELS.geErrorCardChargedBDIFailed,
+            multilineMessages: [{
+                message: apexException.errorMessage || this.CUSTOM_LABELS.commonUnknownError,
+                index: 0
+            }]
+        }];
+        this.hasPageLevelError = true;
```
Comment (psadie-sfdo) [inline src/lwc/geFormRenderer/geFormRenderer.js:370]: @dfuller114 - pinging you to get your feedback around this usage of our multiline form errors

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HuOLQA0" target="_blank">W-040613</a>

**Comment by melissabarber** [comment]
@psadie-sfdo and @ptewson-sfdo I'm signing off on this PR. Thanks!

**Pull request merged**